classiclasagne.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div>
+        <h1><a href="./index.html">Home</a></h2>
+    </div>
+    <h2>Easy classic lasagne</h2>
+    <h2>Ingredients</h2>
+    <ul>
+      <li>1 tbsp olive oil</li>
+      <li>2 rashers smoked streaky bacon</li>
+      <li>1 onion , finely chopped</li>
+      <li>1 celery stick, finely chopped</li>
+      <li>1 medium carrot , grated</li>
+    </ul>
+    <h2>Preparation time</h2>
+    <ul>
+      <li>60 mins</li>
+    </ul>
+    <h2>Instructions</h2>
+    <ul>
```

Comment:
What is the purpose of the unordered list on all files, if it does not contain any li tags?